pkg/controllers/scheduling/scheduling_controller.go

```diff
@@ -147,28 +150,54 @@ func NewSchedulingController(
 		}, placementDecisionInformer.Informer()).
 		WithBareInformers(clusterInformer.Informer(), clusterSetInformer.Informer(), clusterSetBindingInformer.Informer()).
 		WithSync(c.sync).
+		ResyncEvery(ResyncInterval).
 		ToController(schedulingControllerName, recorder)
 }
 
 func (c *schedulingController) sync(ctx context.Context, syncCtx factory.SyncContext) error {
 	queueKey := syncCtx.QueueKey()
-	namespace, name, err := cache.SplitMetaNamespaceKey(queueKey)
-	if err != nil {
-		// ignore placement whose key is not in format: namespace/name
-		utilruntime.HandleError(err)
-		return nil
-	}
-
 	klog.V(4).Infof("Reconciling placement %q", queueKey)
-	placement, err := c.placementLister.Placements(namespace).Get(name)
-	if errors.IsNotFound(err) {
-		// no work if placement is deleted
+
+	if queueKey == "key" {
+		placements, err := c.placementLister.List(labels.Everything())
+		if err != nil {
+			return err
+		}
+
+		for _, placement := range placements {
+			for _, config := range placement.Spec.PrioritizerPolicy.Configurations {
+				if config.ScoreCoordinate != nil && config.ScoreCoordinate.Type == clusterapiv1alpha1.ScoreCoordinateTypeAddOn {
+					key := fmt.Sprintf("%s/%s", placement.Namespace, placement.Name)
```

Comment:
nit there is a helper fun in cache package to generate key

Reply:
fixed.